components/ToDo/index.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import {
+  InputField,
+  ToDoBody,
+  AddButton,
+  DeleteButtonToDo,
+  ToDoBackground,
+  ToDoP,
+  InputPosition,
+} from "./index.styled";
+
+export default function ToDo() {
+  const [todos, setTodos] = useState([]);
+  const [task, setTask] = useState("");
+
+  function addTodo() {
+    if (task !== "") {
+      const newTodo = {
+        id: Date.now(),
```

Comment:
You've probably seen this before =)
```suggestion
        id: Math.random().toString(36).substring(2),
```